src/game/overlay.c

```diff
@@ -78,12 +80,90 @@ static RGB888 m_ColorBarMap[][COLOR_STEPS] = {
       { 120, 60, 70 } },
 };
 
+typedef struct BAR_INFO {
+    int32_t type;
```

Comment:
Consider making it a `BAR_TYPE`

Reply:
I originally had it as that, but then it became `m_HealthBar.bar_type` which seemed like unneeded characters.